app/javascript/packs/src/settings/components/account/preferences/NotificationsSwitchGroup.jsx

```diff
@@ -64,7 +64,7 @@ class NotificationsSwitchGroup extends Component {
         break;
       default:
         this.setState({
-          isSciNoteSwitchOn: true,
+          isSciNoteSwitchOn: false,
```

Comment:
funny naming :smile:

Reply:
Yeah. I know. Maybe

isSwitchOn is better?

Reply:
maybe :smile: